File: java/Scanner.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scn = new Scanner (System.in);

        int lol = scn.nextInt();        

        for(int i = 1; i <= lol; i++) {
            
            if (i % 5 == 0 && i % 3 == 0) {
                System.out.println(i + " fizzbuzz");
            } else if (i % 5 == 0) {
                System.out.println(i +" buzz");
            } else if (i % 3 == 0) {
                System.out.println(i + " fizz");
            } else {
                System.out.println(i);
            }
        }
    }
}

File: javaScript/scanner.js
length = process.argv[2]
flagOne = process.argv[3]
flagTwo = process.argv[4]

out = ""

for (i = 1; i <= length; i++) {
    if (i % flagOne == 0 && i % flagTwo == 0) {
        out += "FizzBuzz "
    }
    else if(i % flagOne == 0) {
        out += "Fizz "
    }
    else if (i % flagTwo == 0) {
        out += "Buzz "
    }
    else {
        out += i + " "
    }
}

console.log(out)